### PR TITLE
Abstract and reference annotations

### PR DESCRIPTION
**Pull request by jjc2718**
For now, I just have the reference annotations in a text file.

We could add the annotations as a section before the references (as suggested in manubot/rootstock#271) or just wait until submission before deciding what to do with them. Either is fine with me.

### CODE REVIEW

File: content/01.abstract.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 ## Abstract {.page_break_before}
-
+In biomedical applications of machine learning, it is often the case that relevant information has a rich structure that is not easily encoded as real-valued predictors.
```
Comment (cgreene) [inline content/01.abstract.md]:
```suggestion
In biomedical applications of machine learning, relevant information often has a rich structure that is not easily encoded as real-valued predictors.
```

File: content/01.abstract.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+Examples of such structured data include DNA or RNA sequence, gene sets or pathways, gene interaction or coexpression networks, ontologies, and phylogenetic trees.
```
Comment (cgreene) [inline content/01.abstract.md]:
```suggestion
Examples of such data include DNA or RNA sequences, gene sets or pathways, gene interaction or coexpression networks, ontologies, and phylogenetic trees.
```

File: content/01.abstract.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
...
+In this review, we highlight recent examples of machine learning models that incorporate structured data into model training or use structure to constrain model architecture.
```
Comment (cgreene) [inline content/01.abstract.md]:
```suggestion
We highlight recent examples of machine learning models that use structure to constrain model architecture or incorporate structured data into model training.
```

File: content/01.abstract.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
...
+For future machine learning applications in biomedicine, particularly when sample size is limited or model interpretability is critical, incorporating prior knowledge in the form of structured data will remain a useful approach.
```
Comment (cgreene) [inline content/01.abstract.md]:
```suggestion
For machine learning in biomedicine, where sample size is limited and model interpretability is critical, incorporating prior knowledge in the form of structured data can be particularly useful.
```

File: content/01.abstract.md [hunk 5]
```diff
@@ -1,3 +1,7 @@
...
+We foresee continued innovation and critical evaluation in this area going forward.
```
Comment (cgreene) [inline content/01.abstract.md]:
```suggestion
The area of research would benefit from performant open source implementations and independent benchmarking efforts.
```

**Comment by cgreene** [comment]
I also went through the ref annotation comments now and they look fine to me 👍

File: content/reference-annotations.txt [hunk 6]
```diff
@@ -0,0 +1,106 @@
+Reference Annotations
+=====================
```
Comment (dhimmel) [inline content/reference-annotations.txt]: Name file something like `content/80.reference-annotations.md`.

And use the actual citations.

It should be just like any other section of the manuscript.

File: content/reference-annotations.txt [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+
+Sequence models:
+
+[**]: BPNet (https://doi.org/10.1101/737981)
```
Comment (dhimmel) [inline content/reference-annotations.txt]: Why not:

```suggestion
Annotation for @doi:10.1101/737981:
```

Comment (dhimmel) [reply content/reference-annotations.txt]: Or `Annotation for BPNet [@doi:10.1101/737981]`

**Pull request merged**